elasticdl/proto/elasticdl.proto

```diff
@@ -140,13 +140,18 @@ message ReportEvaluationMetricsRequest {
     Tensor labels = 3;
 }
 
+message ReportVersionRequest {
+    int32 model_version = 1;
+}
+
 service Master {
     rpc GetTask(GetTaskRequest) returns (Task);
     rpc GetModel(GetModelRequest) returns (Model);
     rpc ReportVariable(ReportVariableRequest) returns (google.protobuf.Empty);
     rpc ReportGradient(ReportGradientRequest) returns (ReportGradientResponse);
     rpc ReportEvaluationMetrics(ReportEvaluationMetricsRequest) returns (ReportEvaluationMetricsResponse);
     rpc ReportTaskResult(ReportTaskResultRequest) returns (google.protobuf.Empty);
+    rpc report_version(ReportVersionRequest) returns (google.protobuf.Empty);
```

Comment:
report_version -> ReportVersion to be consistent with others?

Reply:
Done